### PR TITLE
Add label to configuration file

### LINKED ISSUE #99: Error in generating IL for singleton pattern
I was trying to generate code for a simple singleton implementation 

```csharp
public sealed class Foo {  
    private Foo() {}  
    private static Foo instance = null;  
    public static Foo Instance {  
        get {  
            if (instance == null) {  
                instance = new Foo();  
            }  
            return instance;  
        }  
    }  
}
```

and got the attached error:

```
Something went wrong. Please report the following error in the google group or in the git repository: 
System.Collections.Generic.KeyNotFoundException: The given key 'Label' was not present in the dictionary. at 
System.Collections.Generic.Dictionary`2.get_Item(TKey key) at Cecilifier.Core.Naming.DefaultNameStrategy.PrefixFor(ElementKind 
kind) in /home/adriano/Development/Adriano/study/DotNet/Cecilifier/Cecilifier.Core/Naming/DefaultNameStrategy.cs:line 70 at 
Cecilifier.Core.Naming.DefaultNameStrategy.Label(String name) in /home/adriano/Development/Adriano/study/DotNet/Cecilifier/
Cecilifier.Core/Naming/DefaultNameStrategy.cs:line 58 at Cecilifier.Core.AST.IfStatementVisitor.VisitIfStatement(
IfStatementSyntax node) in /home/adriano/Development/Adriano/study/DotNet/Cecilifier/Cecilifier.Core/AST/
IfStatementVisitor.cs:line 24 at Microsoft.CodeAnalysis.CSharp.Syntax.IfStatementSyntax.Accept(CSharpSyntaxVisitor visitor) at 
Cecilifier.Core.AST.IfStatementVisitor.Visit(IVisitorContext context, String ilVar, IfStatementSyntax node) in /home/adriano/
Development/Adriano/study/DotNet/Cecilifier/Cecilifier.Core/AST/IfStatementVisitor.cs:line 17 at 
Cecilifier.Core.AST.StatementVisitor.VisitIfStatement(IfStatementSyntax node) in /home/adriano/Development/Adriano/study/DotNet/
Cecilifier/Cecilifier.Core/AST/StatementVisitor.cs:line 159 at Microsoft.CodeAnalysis.CSharp.Syntax.IfStatementSyntax.Accept(
CSharpSyntaxVisitor visitor) at Microsoft.CodeAnalysis.CSharp.CSharpSyntaxWalker.Visit(SyntaxNode node) at 
Cecilifier.Core.AST.StatementVisitor.Visit(SyntaxNode node) in /home/adriano/Development/Adriano/study/DotNet/Cecilifier/
Cecilifier.Core/AST/StatementVisitor.cs:line 31 at Microsoft.CodeAnalysis.CSharp.CSharpSyntaxWalker.DefaultVisit(SyntaxNode 
node) at Microsoft.CodeAnalysis.CSharp.CSharpSyntaxVisitor.VisitBlock(BlockSyntax node) at 
Microsoft.CodeAnalysis.CSharp.Syntax.BlockSyntax.Accept(CSharpSyntaxVisitor visitor) at 
Cecilifier.Core.AST.StatementVisitor.Visit(IVisitorContext context, String ilVar, StatementSyntax node) in /home/adriano/
Development/Adriano/study/DotNet/Cecilifier/Cecilifier.Core/AST/StatementVisitor.cs:line 23 at 
Cecilifier.Core.AST.PropertyDeclarationVisitor.<>c__DisplayClass7_0.g__AddGetterMethod|5(AccessorDeclarationSyntax accessor) in /
home/adriano/Development/Adriano/study/DotNet/Cecilifier/Cecilifier.Core/AST/PropertyDeclarationVisitor.cs:line 256 at 
Cecilifier.Core.AST.PropertyDeclarationVisitor.ProcessPropertyAccessors(BasePropertyDeclarationSyntax node, String 
propertyDeclaringTypeVar, String propName, String propertyType, String propDefVar, List`1 parameters, 
ArrowExpressionClauseSyntax arrowExpression) in /home/adriano/Development/Adriano/study/DotNet/Cecilifier/Cecilifier.Core/AST/
PropertyDeclarationVisitor.cs:line 127 at Cecilifier.Core.AST.PropertyDeclarationVisitor.VisitPropertyDeclaration(
PropertyDeclarationSyntax node) in /home/adriano/Development/Adriano/study/DotNet/Cecilifier/Cecilifier.Core/AST/
PropertyDeclarationVisitor.cs:line 73 at Microsoft.CodeAnalysis.CSharp.Syntax.PropertyDeclarationSyntax.Accept(
CSharpSyntaxVisitor visitor) at Microsoft.CodeAnalysis.CSharp.CSharpSyntaxWalker.Visit(SyntaxNode node) at 
Cecilifier.Core.AST.TypeDeclarationVisitor.VisitPropertyDeclaration(PropertyDeclarationSyntax node) in /home/adriano/Development/
Adriano/study/DotNet/Cecilifier/Cecilifier.Core/AST/TypeDeclarationVisitor.cs:line 59 at 
Microsoft.CodeAnalysis.CSharp.Syntax.PropertyDeclarationSyntax.Accept(CSharpSyntaxVisitor visitor) at 
Microsoft.CodeAnalysis.CSharp.CSharpSyntaxWalker.Visit(SyntaxNode node) at 
Microsoft.CodeAnalysis.CSharp.CSharpSyntaxWalker.DefaultVisit(SyntaxNode node) at 
Microsoft.CodeAnalysis.CSharp.CSharpSyntaxVisitor.VisitClassDeclaration(ClassDeclarationSyntax node) at 
Cecilifier.Core.AST.TypeDeclarationVisitor.VisitClassDeclaration(ClassDeclarationSyntax node) in /home/adriano/Development/
Adriano/study/DotNet/Cecilifier/Cecilifier.Core/AST/TypeDeclarationVisitor.cs:line 32 at 
Microsoft.CodeAnalysis.CSharp.Syntax.ClassDeclarationSyntax.Accept(CSharpSyntaxVisitor visitor) at 
Microsoft.CodeAnalysis.CSharp.CSharpSyntaxWalker.Visit(SyntaxNode node) at 
Cecilifier.Core.AST.CompilationUnitVisitor.VisitClassDeclaration(ClassDeclarationSyntax node) in /home/adriano/Development/
Adriano/study/DotNet/Cecilifier/Cecilifier.Core/AST/CompilationUnitVisitor.cs:line 64 at 
Microsoft.CodeAnalysis.CSharp.Syntax.ClassDeclarationSyntax.Accept(CSharpSyntaxVisitor visitor) at 
Microsoft.CodeAnalysis.CSharp.CSharpSyntaxWalker.Visit(SyntaxNode node) at 
Microsoft.CodeAnalysis.CSharp.CSharpSyntaxWalker.DefaultVisit(SyntaxNode node) at 
Microsoft.CodeAnalysis.CSharp.CSharpSyntaxVisitor.VisitCompilationUnit(CompilationUnitSyntax node) at 
Cecilifier.Core.AST.CompilationUnitVisitor.VisitCompilationUnit(CompilationUnitSyntax node) in /home/adriano/Development/Adriano/
study/DotNet/Cecilifier/Cecilifier.Core/AST/CompilationUnitVisitor.cs:line 24 at 
Microsoft.CodeAnalysis.CSharp.Syntax.CompilationUnitSyntax.Accept(CSharpSyntaxVisitor visitor) at 
Microsoft.CodeAnalysis.CSharp.CSharpSyntaxWalker.Visit(SyntaxNode node) at Cecilifier.Core.Cecilifier.Run(Stream content, 
CecilifierOptions options) in /home/adriano/Development/Adriano/study/DotNet/Cecilifier/Cecilifier.Core/Cecilifier.cs:line 42 at 
Cecilifier.Web.Startup.g__CecilifyCode|6_2(WebSocket webSocket) in /home/adriano/Development/Adriano/study/DotNet/Cecilifier/Cecilifier.Web/Startup.cs:line 115

```

### PR DESCRIPTION
**Pull request by Operdies**
I only discovered this project today, so I am not super familiar with how things are structured. I hope this is a workable solution!

Closes #99

### CODE REVIEW

**Comment by Operdies** [comment]
There is also no setting for `ElementKind.StaticConstructor`, but this does not seem to cause issues.

File: Cecilifier.Web/wwwroot/js/cecilifier.js [hunk 1]
```diff
@@ -289,6 +289,7 @@ function initializeSettings(formattingSettingsSample) {
         new Setting(ElementKind.Delegate, {line: startLine + 30, ch: 5}, "Delegate", "delegate prefix","ADelegate", "del"),
         new Setting(ElementKind.LocalVariable, {line: startLine + 32, ch: 5}, "Local Variable", "local variable prefix","i", "lv"),
         new Setting(ElementKind.MemberReference, {line: startLine + 34, ch: 5}, "Member Reference", "Member reference prefix","i", "mr"),
+        new Setting(ElementKind.Label, {line: startLine + 36, ch: 5}, "Jump Label", "Jump Label Prefix","jump", "lbl"),
```
Comment (adrianoc) [inline Cecilifier.Web/wwwroot/js/cecilifier.js:292]: Thanks for the PR

This is the correct fix. It is just missing an update ([in this exact same file](https://github.com/adrianoc/cecilifier/blob/d292a8455b4df2d8ee1734000964be9ed34a6d4c/Cecilifier.Web/wwwroot/js/cecilifier.js#L138)) to the example (showed when you are configuring the prefixes for each element) 

Let me know if you need any help with that.

**Review by adrianoc — CHANGES_REQUESTED** [review]

**Comment by adrianoc** [comment]
> I only discovered this project today, so I am not super familiar with how things are structured. I hope this is a workable solution!
> 
> Closes #99

Not sure it was a coincidence or not... :)
https://twitter.com/adrianoverona/status/1436001076415713285?s=20

File: Cecilifier.Web/wwwroot/js/cecilifier.js (hunk 1 above)
Comment (Operdies) [reply Cecilifier.Web/wwwroot/js/cecilifier.js:294]: Completely unrelated, but I confess that I confess I don't always open PRs on issues I report, but I'm excited about Cecil lately. 

I added an example string in the location you linked. I hope it's okay.

**Review by adrianoc — APPROVED** [review]

**Comment by adrianoc** [comment]
> There is also no setting for `ElementKind.StaticConstructor`, but this does not seem to cause issues.



> There is also no setting for `ElementKind.StaticConstructor`, but this does not seem to cause issues.

Most likely this is not being used :(. I remember that the ctor code delegates to a more generic method that handles methods in general and I never refactored it to be able to use that specific element kind.

**Pull request merged**